mdn-drafts/screens_property.md

```diff
@@ -0,0 +1,46 @@
+---
+recipe: api-interface
+title: 'Screens.screens'
+mdn_url: /en-US/docs/Web/API/Screens/screens
+specifications: https://webscreens.github.io/window-placement/#dom-screens-screens
+browser_compatibility: api.Screens.screens
+---
+
+## Description
+
+The `screens` read-only property of the `Screens` interface returns an array of `ScreenAdvanced` objects which describe the available screens.
```

Comment:
optional nit: "objects, which" or "objects that"

Reply:
Fixed.